woocommerce/woocommerce-gateway-stripe
# Hide/disable UPE when preorders plugin is active

Fixes #1872 

Description: 
Supporting the Pre-Orders plugin with UPE will require more work than we expected. We decided just to hide the UPE option if pre-orders is present. This will also disable UPE if Pre-Orders has been activated later. We already have the function WC_Gateway_Stripe::disable_upe() which basically migrates the UPE settings to regular Stripe settings but it's not possible to call this method directly, instead we update the stripe settings to disable UPE which triggers this method.

# Testing instructions

Enable UPE.
Install and activate [Pre-Orders](https://github.com/woocommerce/woocommerce-pre-orders) plugin
UPE should be disabled and the option to enable it hidden.


-------------------
